packages/web/src/scss/components/Modal/README.md

```diff
@@ -280,6 +282,21 @@ scrolling, e.g.:
 </div>
 ```
 
+### Non-Scrolling Modal Dialog
+
+Scrolling inside ModalDialog can be turned off by adding the `ModalDialog--nonScrolling` modifier class:
+
+```html
+<article class="ModalDialog ModalDialog--nonScrolling">
+  <!-- … -->
+</article>
+```
+
+This way, the ModalBody will expand to fit the height of its content and the whole ModalDialog will scroll in case the
+content is longer than user's viewport.
+
+👉 Please note that this modifier class can produce unexpected results when used with ScrollView.
+
```

Comment:
> 1. Rewrite the styles like the modal has always been non-scrolling [inside].

This is not possible without doing a BC or without a feature flag. Since we want the inside scrolling to be turned on by default anyway, I decided to keep the `ModalDialog--nonScrollable` modifier class. However, I added a deprecation message saying the twist is going to be made in the next major release. (That's why I changed the modifier name from `nonScrolling` to `nonScrollable`—it better matches the future `scrollable` modifier.)